### PR TITLE
Support k8s secrets

### PR DESCRIPTION
**Pull request by ffais**
Added support for kubernetes secrets as mysql root credentials provider.

This [PR](https://github.com/nakamasato/mysql-operator/pull/330) is needed.

### CODE REVIEW

File: charts/mysql-operator/templates/manager-rbac.yaml [hunk 1]
```diff
@@ -10,12 +10,8 @@ rules:
   resources:
   - secrets
   verbs:
-  - create
```
Comment (nakamasato) [inline charts/mysql-operator/templates/manager-rbac.yaml:13]: `create` is necessary.

**Review by nakamasato — COMMENTED** [review]
Thanks for updating! Just one comment!

**Comment by sprasad09** [comment]
@nakamasato can we please merge this PR if looks good. Thanks!

**Comment by nakamasato** [comment]
Sorry for the late reply 🙏  i'm gonna merge it.

**Review by nakamasato — COMMENTED** [review]
we can't use template under crds dir. so let me try fixing it in another pr.

**Comment by nakamasato** [comment]
I've updated with https://github.com/nakamasato/helm-charts/pull/25 sorry for taking so long 🙏

**Pull request closed without merging**